src/main/webapp/index.html

```diff
@@ -8,13 +8,38 @@
   </head>
   <body>
     <header>
-      <h1>smallcity</h1>
+      <h1 class="logo">smallcity</h1>
+      <div class="zipcode-container">
+        your zipcode: 
+      </div>
     </header>
 
     <div class="main-container">
       <div class="map-container">
       </div>
       <div class="results-container">
+        <div class="results-card">
+          <div class="results-image">
```

Comment:
This won't work once you start using dynamic content. I suggest using javascript to create these page elements, and assign the correct details. Similar to what you did in your portfolio project.

Reply:
Yes that's the plan, I just had some in there as placeholders to get the feel for the page.